src/Razor/Microsoft.AspNetCore.Razor.Language/src/Extensions/DesignTimeDirectiveTargetExtension.cs

```diff
@@ -22,7 +22,7 @@ public void WriteDesignTimeDirective(CodeRenderingContext context, DesignTimeDir
             {
                 if (node.Children[i] is DirectiveTokenIntermediateNode n)
                 {
-                    WriteDesignTimeDirectiveToken(context, n);
+                    WriteDesignTimeDirectiveToken(context, node, n, i);
```

Comment:
Nit: Can we give these variables clearer names. The `n, i` is a little trippy.

Reply:
@captainsafia agree, VS auto naming :)

Reply:
updated with better naming